doc/crud.md

```diff
@@ -40,6 +55,10 @@ CDTDatastore *ds = [manager datastoreNamed:@"my_datastore"
 CDTDatastore *ds2 = [manager datastoreNamed:@"other_datastore"
                                       error:&error];
 ```
+```swift
+let ds = try manager.datastoreNamed("my_datastore")
+let ds2 = try manager.datastoreNamed("other_datastore")
```

Comment:
To confirm, using `try` in the example is enough to make it obvious the developer using it will need to catch errors somewhere? (i.e., there's no need for a `catch` in the example).